MapboxGLAndroidSDK/src/main/java/com/mapbox/mapboxsdk/http/HttpRequestUrl.java

```diff
@@ -22,20 +24,20 @@ private HttpRequestUrl() {
    * @return the adapted resource url
    */
   public static String buildResourceUrl(@NonNull String host, String resourceUrl, int querySize, boolean offline) {
+    requestBuilder.delete(0, requestBuilder.length());
```

Comment:
We could also use `requestBuilder.setLength(0)` to reset the StringBuilder.

Reply:
ah cool, wasn't aware that this is a better approach (ref https://stackoverflow.com/a/30190425).